src/nectargraphenebase/account.py

```
@@ -627,16 +835,74 @@ def child(self, offset256):
+
+        Returns:
+            PublicKey: A PublicKey (compressed form) constructed from the derived public key bytes.
+
+        Raises:
+            ImportError: If the `ecdsa` library is not available.
+        """
         privkey = PrivateKey(privkey, prefix=prefix or Prefix.prefix)
         secret = unhexlify(repr(privkey))
+
+        if not ECDSA_AVAILABLE:
+            raise ImportError("ecdsa library is required for from_privkey method")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)